avm/utl/types/avm-common-types/CHANGELOG.md

```diff
@@ -0,0 +1,79 @@
+# Changelog
+
+The latest version of the changelog can be found [here](/Azure/bicep-registry-modules/blob/main/avm/utl/types/avm-common-types/CHANGELOG.md).
+
+## 0.5.1
+
+### Changes
+
+- When introducing the `resourceGroupResourceId` property, the `resourceGroupName` property was not removed from the `privateEndpointMultiServiceType` and  privateEndpointSingleServiceType` types.
+
+### Breaking Changes
+
+- None
+
+## 0.5.0
+
+### Changes
+
+- Adds `resourceGroupResourceId` to the private endpoint type
+
+### Breaking Changes
+
+- None
+
+## 0.4.0
+
+### Changes
+
+- Change autorotation parameter name from `autorotationDisabled `to `autorotationEnabled`
+
+### Breaking Changes
+
+- None
+
+## 0.3.0
+
+### Changes
+
+- Added additional variant of CMK UDT for resources supporting auto-rotation
+
+### Breaking Changes
+
+- None
+
+## 0.2.1
+
+### Changes
+
+- Common-Types: Fixed incorrect parameter description: The FQDN parameter is optional, yet the description said it was required.
+- Diverse: Fixed metadata descriptions
+- Fixed static test validating parameter descriptions
+- It checked for a line that looke like ('Required. even though the value of the $description variable at the time only has the description's value, that is Required.
+- Added test that tests the reverse, that is, that a parameter's description starts with Required. in its title if it is required
+
+### Breaking Changes
+
+- None
+
+## 0.2.0
+
+### Changes
+
+- Added missing `export()` annotation
+
+### Breaking Changes
+
+- None
+
+## 0.1.0
+
+### Changes
+
+- Initial version
+- Fixed workflow configuration
+- Implemented publish-triggering module change
```

Comment:
Minor but I'd suggest to only have Initial version in the first version changes. Not as an enforcement, but does not make sense to list changes there (anything is a change as the module was not there before 0.1.0).
```suggestion
```

Reply:
@AlexanderSehr agree, @ReneHezser and I also discussed the same on another PR and agreed that only changes to main templates should be added (also readme/test updates/PSRule exeptions should be avoided). We agreed that changelogs are primarily useful for module consumers and only changes qualifyinf for a version update should be added. Would you agree?